bottles/frontend/views/bottle_details.py

```diff
@@ -102,14 +100,16 @@ def __init__(self, details, config, **kwargs):
         self.window = details.window
         self.manager = details.window.manager
         self.config = config
+        self.config["Session_Args"] = ""
+        self.config["Check_Terminal"] = False
```

Comment:
Problem with this is that you are putting these values in the config file, which means they stay persistent after relaunches. What we should be doing is to keep these values per bottle per session, until Bottles is closed.

Reply:
From my understanding, bottle specfic parameters have to be loaded from the config. If we try to set them direcly as varaibles in bottles, they get set for all the bottles. What i am doing here shouldn't be an issue because yes the arguments will saved in the config file but everytime bottles is reopened they get set to default values so in a way they only last for a single session.

Reply:
https://github.com/bottlesdevs/Bottles/blob/4966833e3e115b527d2037874f1b47632961f704/bottles/frontend/views/bottle_details.py#L279

here also, Latest_Executables are stored in config even tho that variable is session sepcfic

Reply:
Alright i figured out what issue you were talking about, i have fixed it, pushing the changes

Reply:
@3c8948b should fix the issue